datadog_checks_dev/tests/tooling/test_utils.py

```diff
@@ -38,6 +39,12 @@ def test_is_logs_only(get_root):
     assert is_logs_only('flink')
 
 
+@mock.patch('datadog_checks.dev.tooling.utils.get_root')
+def test_has_process_signature(get_root):
+    get_root.return_value = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../..'))
```

Comment:
Would be nice to move this path into a named constant for clarity and for reuse (with the test right about it), it's not obvious at first glance that this is pointing at the repo's root.